apps/ejabberd/src/mongoose_metrics_hooks.erl

```diff
@@ -271,13 +271,13 @@ privacy_list_push(Acc, _From, #jid{server = Server} = _To, _Broadcast, SessionCo
 user_ping_timeout(Acc, _JID) ->
     Acc.
 
--spec privacy_check_packet(Acc :: allow | deny | block,
+-spec privacy_check_packet(Acc :: mongoose_acc:t(),
                           binary(),
                           Server :: ejabberd:server(),
-                          term(), term(), term()) -> allow | deny | block.
+                          term(), term(), term()) -> mongoose_acc:t().
 privacy_check_packet(Acc, _, Server, _, _, _) ->
     mongoose_metrics:update(Server, modPrivacyStanzaAll, 1),
-    case Acc of
+    case mongoose_acc:get(privacy_check, Acc) of
```

Comment:
I can imagine a situation that `mod_privacy` and `mod_blocking` are disabled. To my understanding in such a case there will be no `privacy_check` at this point which will end up with `case clause` error.